CHANGES.md

```diff
@@ -1,3 +1,20 @@
+## edge-20.1.3
+
+* CLI
+  * Introduced `linkerd check --cni` to ensure the CNI plugins can properly be
+    installed on the cluster
```

Comment:
This ended up being `linkerd check --pre --linkerd-cni-enabled`. So this should read instead something like
```
Introduced `linkerd-check --pre --linkerd-cni-enabled`, used when the CNI plugin is used,  to check it has been properly installed before proceeding with the control plane installation
```

Reply:
Good catch! FWIW, it also works with `linkerd check --linkerd-cni-enabled`.

Reply:
Okay thanks for clarifying!

Reply:
@ihcsim apparently not:
```bash
$ bin/go-run cli check --linkerd-cni-enabled
Error: Validation error when executing check command: --linkerd-cni-enabled can only be used with --pre
```

Reply:
right.  the post-install check will read the CNI mode from the configmap.  but for `--pre` the Linkerd configmap doesn't exist yet so it needs the `--linkerd-cni-enabled` flag